zipline/finance/performance/tracker.py

```diff
@@ -345,7 +365,24 @@ def process_event(self, event):
 
             self.all_benchmark_returns[midnight] = event.returns
 
-    def check_upcoming_dividends(self, midnight_of_date_that_just_ended):
+    def _next_trading_day(self, midnight_of_date_that_just_ended):
```

Comment:
This method (along with a whole host of date manipulation utilities) already exists on `TradingEnvironment`.